ci-operator/step-registry/idp/openldap/idp-openldap-commands.sh

```diff
@@ -184,56 +222,44 @@ EOF
 function configure_identity_provider() {
     echo "Configuring OpenShift identity provider with OpenLDAP"
 
-    # current generation
-    gen=$(oc get deployment oauth-openshift -n openshift-authentication -o jsonpath='{.metadata.generation}')
-
-    # Prepare the new LDAP identity provider configuration as a JSON object
-    new_ldap_idp='{
-        "ldap": {
-            "attributes": {
-                "email": ["mail"],
-                "id": ["dn"],
-                "name": ["cn"],
-                "preferredUsername": ["cn"]
-            },
-            "bindDN": "",
-            "bindPassword": {
-                "name": ""
-            },
-            "insecure": true,
-            "url": "ldap://ldap.ldap.svc:389/ou=people,ou=rfc2307,dc=example,dc=com?cn"
-        },
-        "mappingMethod": "claim",
-        "name": "ldapidp",
-        "type": "LDAP"
-    }'
-
-    # Since no IDP is configured, create a new array with the LDAP IDP
-    combined_idp_config="[$new_ldap_idp]"
-
-    # Construct the JSON patch using a here-document
-    json_patch=$(cat <<EOF
-{
-  "spec": {
-    "identityProviders": $combined_idp_config
-  }
-}
+    # Create network policy to allow access from hypershift namespace to ldap namespace
+    if [ "$IS_HYPERSHIFT_ENV" == "yes" ]; then
+        cat <<EOF | oc apply -f -
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: allow-all-ingress-egress
+  namespace: $OAUTH_NAMESPACE
```

Comment:
CC @heliubj18 in case you have other comment about creating this networkpolicy which allows pods of namespace clusters-<hostedcluster> to be able to reach pods of ldap namespace on the management cluster.